nipype/interfaces/camino/convert.py

```diff
@@ -463,7 +463,8 @@ def _get_actual_outputroot(self, outputroot):
     def _list_outputs(self):
         outputs = self.output_spec().get()
         outputs['proc'] = os.path.abspath(self._gen_outfilename())
-        outputs['outputroot_files'] = self.outputroot_files
+        if isdefined(self.inputs.outputroot):
+            outputs['outputroot_files'] = self.outputroot_files
```

Comment:
I think there's a slightly bigger bug, which is that `self.outputroot_files` is not defined until the interface is run, and then only sometimes. I would instead make sure that it is defined at `__init__`:

```Python
def __init__(self, *args, **kwargs):
    super(ProcStreamlines, self).__init__(*args, **kwargs)
    self.outputroot_files = []
```

This way `_list_outputs()` will not raise an exception either before or after the interface is run, and whether `self.inputs.outputroot` is defined or not. For `OutputMultiPath`s, `[]` is equivalent to `Undefined`, so this change can be reverted, as well.